README.md

```diff
@@ -1,52 +1,86 @@
-Yunohost app for « I hate money » budget web app
-================================================
+# I Hate Money for YunoHost
 
-[![Install « I hate money » with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=ihatemoney)
+[![Integration level](https://dash.yunohost.org/integration/ihatemoney.svg)](https://dash.yunohost.org/appci/app/ihatemoney) ![](https://ci-apps.yunohost.org/ci/badges/ihatemoney.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/ihatemoney.maintain.svg)  
+[![Install ihatemoney with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=ihatemoney)
 
-- Supported Yunohost versions : 2.6.x, 2.7.x 3.x
-- Tested Yunohost version : 3.3.1
+*[Lire ce readme en français.](./README_fr.md)*
 
-*NB: That means I'll try not to drop support for YunoHost 2.x too soon, and
-accept patches to keep retro-compatibility, but I'll not test it myself against
-YunoHost 2.x*
+> *This package allows you to install I Hate Money quickly and simply on a YunoHost server.  
+If you don't have YunoHost, please consult [the guide](https://yunohost.org/#/install) to learn how to install it.*
+
+## Overview
+I Hate Money is a web application made to ease shared budget management. It keeps track of who bought what, when, and for whom; and helps to settle the bills.
+
+**Shipped version:** 4.1.3
```

Comment:
Shipped version is specified a loosy way, it is actually 4.x.

Reply:
I think we should be specific about delivered version because users want to know which version is provided. Let's make life easy :)

Reply:
Yes, that is a good discussion. [Current behavior](https://github.com/YunoHost-Apps/ihatemoney_ynh/blob/master/scripts/_common.sh#L46) does not target a specific version but 4.x.

So I'm open to discussion and change on that.
But this is a documentation PR, and I think documentation must document what is actually there, not a dreamed situation… And evolve along with the software evolutions :-).